design-proposals/customize-dns-probe-globally.md

```diff
@@ -0,0 +1,62 @@
+# Globally customize DNS cluster health probe.
+
+## Summary
+
+Support customize kubernetes-nmstate DNS resolving probes at NMState CR.
+
+## Motivation
```

Comment:
Please also list the alternatives in this document. e.g. making the probe smarter, and having this configured on NNCP-level

Reply:
I will add an alternative about probe doing fallaback to a global name, about NNCP is stated as non goal and if needed can be done a new enhancement, we want this one to be very specific to reduce the number of knobs that we really need.